### PR TITLE
[blitz] Add new port

### PR DESCRIPTION
**Pull request by NancyLi1013**
This port doesn't currently support `arm`, `arm64 `and `uwp`.

Related issue #1661.

No feature need to test.

### CODE REVIEW

File: ports/blitz/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,37 @@
+include(vcpkg_common_functions)
```
Comment (Neumann-A) [inline ports/blitz/portfile.cmake]: `vcpkg create` again. This line is no longer necessary

File: ports/blitz/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,37 @@
+include(vcpkg_common_functions)
+
+if(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm" OR VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64")
```
Comment (Neumann-A) [inline ports/blitz/portfile.cmake]: this should be `vcpkg_fail_port_install`

Comment (NancyLi1013) [reply ports/blitz/portfile.cmake]: Thanks a lot for your review. Have updated.

File: ports/blitz/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+vcpkg_install_cmake()
+
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake)
+
+# Handle copyright
+file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT})
+file(RENAME ${CURRENT_PACKAGES_DIR}/share/${PORT}/LICENSE ${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright)
```
Comment (JackBoosY) [inline ports/blitz/portfile.cmake]: Should be:
`file(RENAME ${CURRENT_PACKAGES_DIR}/share/${PORT}/LICENSE ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright) `

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
The current failures are on `x64-uwp`, `arn64-windows`, `arm-uwp`, which are as expected.

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**